gtk4/Gir.toml

```diff
@@ -503,6 +501,9 @@ manual_traits = ["BuilderExtManual"]
     name = "get_object"
     manual = true # add dynamic casting
     doc_trait_name = "BuilderExtManual"
+    [[object.function]]
+    name = "value_from_string"
+    manual = true # ParamSpec is a fundemental type
```

Comment:
Why does that matter? You can treat it just like a boxed type and that should be autogeneratable or not?

Reply:
How can we mark ParamSpec as Boxed from gir? i don't think it's possible for now

Reply:
That's probably true :)

Reply:
It's, what would be nice is to make Gir understand the difference between an object and a fundemental and just not use `IsA` for fundemental types.

Reply:
Yes, please create an issue for that

Reply:
https://github.com/gtk-rs/gir/issues/1069